include/ignition/math/Helpers.hh

```diff
@@ -727,8 +729,53 @@ namespace ignition
       }
     }
 
+    /// \brief Convert a std::chrono::system_clock::time_point to a seconds and
+    /// nanoseconds pair.
+    /// NOTE: On gcc, system_clock::time_point has nanosecond precision,
+    // on Windows, it has precision 1/10'000'000 (100ns),
+    // and on macOS, microsecond precision.
+    /// \param[in] _time The time point to convert.
+    /// \return A pair where the first element is the number of seconds and
+    /// the second is the number of nanoseconds.
+    inline std::pair<int64_t, int64_t> timePointToSecNsec(
+        const std::chrono::system_clock::time_point &_time)
```

Comment:
I think that we want to use `steady_clock` which represents monotonic time that cannot be disrupted by daylight savings or any other sort of time-weirdness.

Reply:
https://en.cppreference.com/w/cpp/chrono/steady_clock

Reply:
Changed to steady_clock 

https://github.com/ignitionrobotics/ign-math/pull/150/commits/8067150c00066316eab3fb3b0e0264e9c2d67218